commons/all/src/main/java/org/infinispan/commons/api/BasicCache.java

```diff
@@ -12,25 +12,20 @@
 /**
  * BasicCache provides the common building block for the two different types of caches that Infinispan provides:
  * embedded and remote.
- * <p/>
- * For convenience, BasicCache extends {@link ConcurrentMap} and implements all methods accordingly, although methods like
+  * For convenience, BasicCache extends {@link ConcurrentMap} and implements all methods accordingly, although methods like
  * {@link ConcurrentMap#keySet()}, {@link ConcurrentMap#values()} and {@link ConcurrentMap#entrySet()} are expensive
  * (prohibitively so when using a distributed cache) and frequent use of these methods is not recommended.
- * <p />
  * Other methods such as {@link #size()} provide an approximation-only, and should not be relied on for an accurate picture
  * as to the size of the entire, distributed cache.  Remote nodes are <i>not</i> queried and in-fly transactions are not
  * taken into account, even if {@link #size()} is invoked from within such a transaction.
- * <p/>
- * Also, like many {@link ConcurrentMap} implementations, BasicCache does not support the use of <tt>null</tt> keys or
+  * Also, like many {@link ConcurrentMap} implementations, BasicCache does not support the use of <code>null</code> keys or
  * values.
- * <p/>
- * <h3>Unsupported operations</h3>
+  * <h2>Unsupported operations</h2>
  * <p>{@link #containsValue(Object)}</p>
  *
  * Please see the <a href="http://www.jboss.org/infinispan/docs">Infinispan documentation</a> and/or the <a
  * href="https://docs.jboss.org/author/display/ISPN/Getting+Started+Guide#GettingStartedGuide-5minutetutorial">5 Minute Usage Tutorial</a> for more details.
- * <p/>
```

Comment:
The url above doesn't resolve anymore btw, if we want to remove it.

Reply:
Removed